protocols/onboarding.md

```diff
@@ -69,5 +69,8 @@ Be assured that Kevin has aggressively curated his notifications 😉.
 
 [zulip]: https://bonhamlab.zulipchat.com/
 
+<br>
+<br>
```

Comment:
```suggestion
~~~
<br>
<br>
~~~
```

For this SSG, if you want to add literal HTML, you have to wrap it in triple tilde like this